### ISSUE TITLE
When a panic occurs in a Task within a subflow, should the entire workflow be interrupted?

### ISSUE THREAD

**Issue by Eviltuzki** [noneback/go-taskflow#53]
I think there should be at least an option or method provided to let users decide whether to interrupt or continue execution.

code example such as:
```
package main

import (
	"fmt"
	gotaskflow "github.com/noneback/go-taskflow"
	"time"
)

func main() {
	executor := gotaskflow.NewExecutor(100000)
	tf := gotaskflow.NewTaskFlow("G")
	copy_gcc_source_file := tf.NewTask("copy_gcc_source_file", func() {
		time.Sleep(1 * time.Second)
		fmt.Println("copy_gcc_source_file")
	})
	tar_gcc_source_file := tf.NewTask("tar_gcc_source_file", func() {
		time.Sleep(1 * time.Second)
		fmt.Println("tar_gcc_source_file")
	})
	download_prerequisites := tf.NewSubflow("download_prerequisites", func(sf *gotaskflow.Subflow) {
		sf.NewTask("", func() {
			time.Sleep(1 * time.Second)
			fmt.Println("download_prerequisites")
			panic(1)
		})
	})
	yum_install_dependency_package := tf.NewTask("yum_install_dependency_package", func() {
		time.Sleep(1 * time.Second)
		fmt.Println("yum_install_dependency_package")
	})
	mkdir_and_prepare_build := tf.NewTask("mkdir_and_prepare_build", func() {
		time.Sleep(1 * time.Second)
		fmt.Println("mkdir_and_prepare_build")
	})
	make_build := tf.NewTask("make_build", func() {
		time.Sleep(1 * time.Second)
		fmt.Println("make_build")
	})
	make_install := tf.NewTask("make_install", func() {
		time.Sleep(1 * time.Second)
		fmt.Println("make_install")
	})
	relink := tf.NewTask("relink", func() {
		time.Sleep(1 * time.Second)
		fmt.Println("relink")
	})
	copy_gcc_source_file.Precede(tar_gcc_source_file)
	yum_install_dependency_package.Precede(download_prerequisites)
	tar_gcc_source_file.Precede(download_prerequisites)
	download_prerequisites.Precede(mkdir_and_prepare_build)
	mkdir_and_prepare_build.Precede(make_build)
	make_build.Precede(make_install)
	make_install.Precede(relink)
	executor.Run(tf).Wait()
	//fmt.Println("Print DOT")
	//if err := tf.Dump(os.Stdout); err != nil {
	//	log.Fatal(err)
	//}
}
```

**Comment by noneback** [comment]
That makes sense. We can add a config to decide whether tf should continue to execute when panic occurs.

**Comment by noneback** [comment]
I also tried your example on my PC. It didn't stop scheduling as we expected when panic occurred. I think this is a bug, and I fix it in #54
